src/frontends/tensorflow/tests/compilation.cpp

```diff
@@ -32,7 +32,7 @@ std::shared_ptr<ov::Model> convert_model(const std::string& model_path) {
 
 class CompileModelsTests : public ::testing::Test {};
 
-TEST_F(CompileModelsTests, NgramCompilation) {
+TEST_F(CompileModelsTests, DISABLED_NgramCompilation) {
     ov::Core core;
     auto model = convert_model("model_ngram/model_ngram.pb");
     ov::CompiledModel compiled_model = core.compile_model(model, "CPU");
```

Comment:
TF tests should not depend on CPU plugin I suppose

Reply:
it should be dependent. Because this covers full conversion pipeline and the tested transformation is highly sensitive to what transformations applied before

Reply:
we need at least add a conditional compilation for cases when CPU plugin is disabled in cmake

Reply:
In the main PR, I did [that](https://github.com/openvinotoolkit/openvino/pull/16131/files#diff-e8ce14587a7e9ca7a3909dcd3ebed67ab885da13adc8b3f83a03e4f080ad89b4). Could you please comment whether it is applicable or we have a better solution?

Reply:
the solution is good.

I supposed to see the condition in cpp files, but in cmake it also works well